src/coord/src/coord.rs

```diff
@@ -2955,6 +2955,33 @@ impl Coordinator {
                                     1,
                                 )
                                 .await;
+                                match connector {
+                                    SinkConnector::Kafka(KafkaSinkConnector { topic, .. }) => {
+                                        let row = Row::pack_slice(&[
+                                            Datum::String(&entry.id().to_string()),
+                                            Datum::String(topic.as_str()),
+                                        ]);
+                                        self.update_catalog_view(
```

Comment:
Another question that predates this PR: why do these sinks have their own system tables?

Reply:
Ah, because we need to expose additional metadata about the sink to the user that varies by sink type. So Kafka sinks produce the topic name but Avro OCF sinks produce the Avro OCF name. You could imagine jamming it all into one table, but these columns are actually of different types (text vs bytes), and so having separate tables makes it all a bit more type safe.